CMakeLists.txt

```diff
@@ -92,7 +92,7 @@ morpheus_utils_initialize_cuda_arch(morpheus)
 # Note intentionally excluding CUDA from the LANGUAGES list allowing us to set some clang specific settings later when
 # we call morpheus_utils_enable_cuda()
 project(morpheus
-  VERSION 24.06.00
+  VERSION 24.06.00rc1
```

Comment:
This shouldn't be needed, and will need to change once we cut the release

Reply:
Reverted to `24.06.00`